arangod/RestHandler/RestVocbaseBaseHandler.cpp

```diff
@@ -584,6 +585,18 @@ std::unique_ptr<SingleCollectionTransaction> RestVocbaseBaseHandler::createTrans
       LOG_TOPIC("e94ea", DEBUG, Logger::TRANSACTIONS) << "Transaction with id '" << tid << "' not found";
       THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_TRANSACTION_NOT_FOUND, std::string("transaction '") + std::to_string(tid) + "' not found");
     }
+    auto* state = ctx->getParentTransaction();
+    TRI_ASSERT(state != nullptr);
+    TRI_voc_cid_t cid = 0;
+    if (state->isCoordinator()) {
```

Comment:
I think we should just check the name against the name string or the ID

Reply:
I don't understand. Can you elaborate?